.github/workflows/server-release.yml

```diff
@@ -42,6 +42,10 @@ jobs:
         if: ${{ matrix.target == 'x86_64-unknown-linux-musl' }}
         run: sudo apt-get install -y musl-dev musl-tools
 
+      - name: Install OpenSSL for Windows
+        if: ${{ matrix.target == 'x86_64-pc-windows-gnu' }}
```

Comment:
Are we using windows-gnu?

Reply:
https://github.com/svix/svix-webhooks/blob/main/.github/workflows/server-release.yml#L15
yep